content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -489,6 +489,40 @@ spec:
         - containerPort: 80
 ```
 
+### Kubelet and kube-proxy can now run as Windows services
+
+Starting with kubernetes v1.11, kubelet and kube-proxy can run as Windows services.
+
+This means that you can now register them as Windows services via `sc` command. More details about how to create Windows services with `sc` can be found [here](https://support.microsoft.com/en-us/help/251192/how-to-create-a-windows-service-by-using-sc-exe).
+
+**Examples:**
```

Comment:
Just curious, why should this be a level 4 heading? All the other features have level 3 heading.

Reply:
@alinbalutoiu Are the examples under `### Kubelet and kube-proxy can now run as Windows services`? If so, it's clearer to format the heading as `#### Examples` rather than just bolding the text.

Reply:
@zacharysarah Yes the examples are under ``### Kubelet and kube-proxy can no run as Windows services``. I wanted to stick with the previous format (example: https://github.com/kubernetes/website/pull/8454/files#diff-10523ec8ea4ab689a6b2b75bf6e3a5c5R363). If you think that way is clearer I can update afterwards to replace with ``#### Examples`` both usages.